helix-term/src/keymap/default.rs

```diff
@@ -368,7 +368,8 @@ pub fn default() -> HashMap<Mode, KeyTrie> {
         "C-h" | "backspace" | "S-backspace" => delete_char_backward,
         "C-d" | "del" => delete_char_forward,
         "C-j" | "ret" => insert_newline,
-        "tab" => insert_tab,
+        "tab" => smart_tab,
+        "S-tab" => insert_tab,
```

Comment:
one more minor ching from my side, I thaught we wanted to add `a-e` and `a-b` default bindings as well? There was a lot of back and forth so maybe I forgot (sorry in that case) . Espeically in select mode I find these bindings quite usefel

Reply:
Oh yeah, I forgot about that. I'll add that too